NEWS

```diff
@@ -9,6 +9,8 @@ Enhancements
 Bug fixes
  - Fix Outlook crashes when modifying the view of a folder
  - Fix server side crash when reading some recurrence appointments
+ - Fix get recipient so we can get it from client data if they do not have an associated
+   sogo user (i.e. groups)
 
```

Comment:
Although I understand that as developer, this file is intended for users, so my proposal goes to (correct me if I'm wrong):

  * Be able to have group mail as to, cc, bcc or from

If we won't want to have merging problems, we can have this information available in the PR and the one who merges it updates the `NEWS` file accordingly.